docs/concepts/add-in-development-lifecycle.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Office Add-ins development lifecycle
```

Comment:
Do we want to put that PowerApps note is this file? It's all about which platform to choose.

Reply:
@AlexJerabek No, I don't think so at this point. PowerApps replace Access Web Apps so where I had to keep the Access reference I pointed users to PowerApps. However, with AWA's deprecation, Access no longer supports add-ins and I can't say that PowerApps do.